dev-libs/capstone/capstone-5.0.1.ebuild

```diff
@@ -90,3 +88,7 @@ src_install() {
 		find "${ED}" -name '*.a' -delete || die
 	fi
 }
+
+python_test() {
+	emake check
```

Comment:
Looking at the Makefile, this isn't going to work, as this target doesn't fail if tests fail.

Reply:
Please don't confuse `Makefile` with `bindings/python/Makefile`. The `check` target in the former one indeed discards all errors (but luckily the checks are executed through CMake as well):

```Makefile
check: $(TESTS)

test_%:
        ./tests/$@ > /dev/null && echo OK || echo FAILED
```

`python_test` is executed in `bindings/python` so the `check` target of the latter `Makefile` applies:

```Makefile
check:
        @for t in $(TESTS); do \
                echo Check $$t ... ; \
                ./$$t > /dev/null; \
                if [ $$? -eq 0 ]; then echo OK; else echo FAILED; exit 1; fi \
        done
```

This target fails if the first check fails.